pom.xml

```diff
@@ -328,13 +325,6 @@
 					<configuration>
 						<skip>${maven.site.plugin.skip}</skip>
```

Comment:
I don't know how much this makes sense from your PoV, but we have: https://maven.apache.org/plugins/maven-site-plugin/site-mojo.html#skip So no need for a custom property...